src/ui/common/src/components/workflows/nodes/NoneArtifactNode.tsx

```diff
@@ -0,0 +1,24 @@
+import { faChartLine } from '@fortawesome/free-solid-svg-icons';
+import React, { memo } from 'react';
+
+import { ReactFlowNodeData } from '../../../utils/reactflow';
+import Node from './Node';
+
+type Props = {
+  data: ReactFlowNodeData;
+  isConnectable: boolean;
+};
+
+const NoneArtifactNode: React.FC<Props> = ({ data, isConnectable }) => {
+  console.log("node data: ", data)
```

Comment:
nit: remove this before merging. 

Do you have a screenshot of this?

Reply:
<img width="1919" alt="image" src="https://user-images.githubusercontent.com/61630725/184978496-35150e52-d4a5-45ed-8c36-04d5f826ba43.png">